src/main/java/com/synopsys/integration/detect/tool/detector/inspectors/ArtifactoryDockerInspectorResolver.java

```diff
@@ -102,6 +102,7 @@ private DockerInspectorInfo install() throws IntegrationException, IOException,
             }
             if (inspector == null) {
                 if (cachedInstall.isPresent()) {
+                    logger.debug("Using docker inspector from previous install.");
```

Comment:
Do we have access to the version at this point? If so `Using Docker Inspector <some-version> from previous install.` would look nice.

Reply:
could try parsing the version from the path/name of the file but otherwise no